util/domd

```diff
@@ -11,7 +11,7 @@ if [ "$1" = "-MD" ]; then
 fi
 if [ "$MAKEDEPEND" = "" ]; then MAKEDEPEND=makedepend; fi
 
-cp Makefile Makefile.save
+cp -p Makefile Makefile.save
```

Comment:
This is troubling.  `-p` is a GNUism, meaning `domd` will break on any machine that doesn't use GNU binutils.